### ISSUE TITLE
Feature Request: Display all peers in peer panel.

### ISSUE THREAD

**Issue by davischw** [transmission/transmission#3987]
### What is the issue?

Currently only peers with active connections are displayed in the peer panel. It would be helpful for users to also have inactive/invalid peers listed with information why they have not been selected as valid peers or why no connection could be established (i.e. protocol errors, peer does not support encryption or no route to peer). Maybe even a right click menu to forcefully set a peer as valid, if user settings prevent from selecting it as a valid peer.

It is very frustrating to have trackers report ~20 peers but only one of them (or none) are selected and the other ones are ignored and the user does not know why, and has no way of influencing/debugging the selection process over the UI.

### Which version of Transmission?

3.00 (bb6b5a062e)

**Comment by Coeur** [comment]
Seems that @GaryElshaw wanted the exact opposite in #2537, so it might need to be a configurable feature.

**Comment by GaryElshaw** [comment]
....different folks....

**Comment by davischw** [comment]
@Coeur There already is a `Show more detail` checkbox to display more columns in the peer panel. An addition of a `Show all peers reported by trackers` checkbox might be acceptable and not unnecessarily clutter the UI. Ticking that box could then also show additional peer connection/validity status columns in the table.

As for the `pr welcome` label; I can try to implement it, but i would first have to get familiar with the codebase and probably require some help of one of the more experienced contributors. That might suck up more personnel resources and take up more time than if one of you guys would do it.

**Comment by GaryElshaw** [comment]
Yep, the 'show more detail' check isn't in the mac client, they visually implement things quite differently.

**Comment by davischw** [comment]
Just installed the `transmission-qt` package. You are right, there are a lot of different knobs to turn between the qt/gtk versions. But ideally, the feature would have to be added to both of them, right?

**Comment by nevack** [comment]
> Just installed the `transmission-qt` package. You are right, there are a lot of different knobs to turn between the qt/gtk versions. But ideally, the feature would have to be added to both of them, right?

Yes, feature parity between different client implementation is one of important project goals.
